modules/juce_audio_plugin_client/LV2/juce_LV2_Wrapper.cpp

```diff
@@ -86,7 +86,7 @@ using namespace juce;
 static const String& getPluginURI()
 {
     // JucePlugin_LV2URI might be defined as a function (eg. allowing dynamic URIs based on filename)
-    static const String pluginURI(JucePlugin_LV2URI);
+    static const String pluginURI { JucePlugin_LV2URI };
```

Comment:
What purpose does this change serve?

Reply:
The original version caused an error on one of the compilers I was building with (I think it was an older gcc...)